projects/cm_mcu/commands/SensorControl.c

```diff
@@ -508,7 +508,96 @@ BaseType_t ff_status(int argc, char **argv, char* m)
   for (; whichff < 25; ++whichff) {
     int8_t status = getFFstatus(whichff);
     const char *name = getFFname(whichff);
-    copied += snprintf(m + copied, SCRATCH_SIZE - copied, "%s %02d", name, status);
+    copied += snprintf(m + copied, SCRATCH_SIZE - copied, "%s %02d ", name, status);
+
+    bool isTx = (strstr(name, "Tx") != NULL);
+    if (isTx)
+      copied += snprintf(m + copied, SCRATCH_SIZE - copied, "\t");
+    else
+      copied += snprintf(m + copied, SCRATCH_SIZE - copied, "\r\n");
+
+    if ((SCRATCH_SIZE - copied) < 20 && (whichff < 25)) {
+      ++whichff;
+      return pdTRUE;
+    }
+  }
+  whichff = 0;
+  return pdFALSE;
+}
+
+BaseType_t ff_los_alarm(int argc, char **argv, char* m) {
+  int copied = 0;
+
+  static int whichff = 0;
+  if (whichff == 0) {
+    copied += snprintf(m + copied, SCRATCH_SIZE - copied, "FIREFLY LOS ALARM:\r\n");
+  }
+  for (; whichff < 25; ++whichff) {
+    const char *name = getFFname(whichff);
+    copied += snprintf(m + copied, SCRATCH_SIZE - copied, "%s ", name);
+    if (!isEnabledFF(whichff)){
```

Comment:
now that you've exported the `isEnabledFF` function can you also update `ff_ctl` to use it?